src/assets/styles/base.sass

```diff
@@ -143,6 +143,10 @@ abbr[title]
 .nav-tabs > li > a:hover
   background-color: #dff0d8 !important
 
+.tab-content
+  border-bottom: 1px solid #ddd
+  margin-bottom: 40px
```

Comment:
```suggestion
.tab-content
  border: solid #ddd
  border-width: 0px 1px 1px 1px
  margin-bottom: 40px
  padding: 10px
  border-radius: 0 0 4px 4px
```

to put it in a box like this:

<img width="866" alt="Screenshot 2023-08-16 at 15 18 43" src="https://github.com/giantswarm/docs/assets/1376043/ba977579-dc6c-4fed-aa21-2ddbbb4bb0d9">